### PR TITLE
Particle render

### PR DESCRIPTION
**Pull request by eyworldwide**
### Please check if the PR fulfills these requirements

- [X] The commit message follows our [guidelines](https://github.com/oasis-engine/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
- Feature: ParticleRenderer add `playOnEnable` and `blendMode` property.
- Bug fix: texture uv Y flip bug. 

### Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)
No.
### Other information:

### CODE REVIEW

File: packages/core/src/particle/ParticleRenderer.ts [hunk 1]
```diff
@@ -380,7 +393,7 @@ export class ParticleRenderer extends GeometryRenderer {
   }
 
   /**
-   * Follow the direction of velocity or not.
+   * Whether follow the direction of velocity.
```
Comment (gz65555) [inline packages/core/src/particle/ParticleRenderer.ts:396]: Error of comments ?

Comment (eyworldwide) [reply packages/core/src/particle/ParticleRenderer.ts]: Fixed.

File: packages/core/src/particle/ParticleRenderer.ts [hunk 2]
```diff
@@ -464,6 +478,47 @@ export class ParticleRenderer extends GeometryRenderer {
...
+  set playOnEnable(value: boolean) {
+    this._playOnEnable = value;
+
+    if (value) {
+      this.start();
+    }
+    else {
+      this.stop();
+    }
+  }
```
Comment (zhuxudong) [inline packages/core/src/particle/ParticleRenderer.ts:497]: Why start/stop() after setting `playOnEnable`. can you use `component.enabled` to replace it?

**Review by gz65555 — APPROVED** [review]
+1

**Pull request merged**